example/service/s3/getObjectWithProgress/getObjectWithProgress.go

```diff
@@ -0,0 +1,162 @@
+package main
```

Comment:
Adding `example` build tag to the top of this fill will prevent it being built under "normal" use cases. We use this to separate the examples from the rest of the SDK when building.

```go
// +build example

package main
```

Reply:
Done! 👍